frontend/src/app/admin/organization/admin-organization.service.ts

```diff
@@ -9,36 +9,45 @@
 
 import { Injectable } from '@angular/core';
 import { HttpClient } from '@angular/common/http';
-import { Organization } from "/workspace/frontend/src/app/organization/organization.service";
-import { Observable } from 'rxjs';
+import { Organization } from "../../..//app/organization/organization.service";

```

Comment:
This path feels off, looks like we only need two levels of traversal up (since both this file and the imported are in frontent/src/app/).